src/layout/Footer/Footer.jsx

```diff
@@ -1,12 +1,34 @@
 import React from "react";
 import * as s from "./Footer.style"
+import { Col, Container, Row } from "react-grid-system";
```

Comment:
The benefit of using row and column, we don't need to manually define the CSS for each breakpoint, simply change the column width for each breakpoint. The column width replaces the lines of CSS, If it is founded to be a better solution, it can be carried out in future sprints, or mentioned in the handover doc for the next round of students.